drivers/ad5592r/ad5592r-base.h

```diff
@@ -91,6 +91,15 @@ struct ad5592r_rw_ops {
 	int32_t (*gpio_read)(struct ad5592r_dev *dev, uint8_t *value);
 };
 
+enum ad5592r_reference {
+	AD5592R_INT_REF_ACTIVE,
+	AD5592R_INT_REF_DEACTIVE
+};
+
+struct ad5592r_init_param {
+	enum ad5592r_reference int_ref;
```

Comment:
I would rather use a boolean variable instead of an enum. Something like `bool use_int_ref;`